setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = lsqecc
+version = 
+description = a Python package for ... 
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://latticesurgery.com/
+author = George Watkins, Alex Nguyen, Varun Seshardi, Keeland Watkins
+author_email = 
+license = 
+license_file = 
+classifiers =
+    Topic :: Scientific/Engineering
+    Development Status :: 3 - Alpha
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Operating System :: OS Independent,
+    Intended Audience :: Science/Research
+    Natural Language :: English
+    Typing :: Typed
+project_urls = 
+    Documentation = 
+    Source = https://github.com/latticesurgery-com/lattice-surgery-compiler
+    Tracker = https://github.com/latticesurgery-com/lattice-surgery-compiler/issues
+
+[options]
+packages = find:
+python_requires = >=3.7
+install_requires =
+    qiskit>=0.31.0
+    pyzx>=0.6.4
+    igraph>=0.9.8
+    mako>=1.1.5
+
+[options.extras_require]
+dev =
```

Comment:
Guess `mypy` will go here as well

Reply:
Yep, but right now it depends on our progress in [mypy-refactor](https://github.com/latticesurgery-com/lattice-surgery-compiler/tree/mypy-refactor).